src/ontology/mondo-edit.obo

```diff
@@ -400975,19 +400959,37 @@ intersection_of: disease_has_location UBERON:0009853 ! body of uterus
 id: MONDO:0021651
 name: synpolydactyly
 def: "A joint presentation of syndactyly (fusion of digits) and polydactyly (production of supernumerary digits)." [https://en.wikipedia.org/wiki/Synpolydactyly]
+subset: gard_rare
+subset: ordo_morphological_anomaly {source="Orphanet:93403"}
 synonym: "polysyndactyly" EXACT [NCIT:C125597, SCTID:84598000, UMLS:C0265553]
+synonym: "SPD" RELATED ABBREVIATION [GARD:0005087]
+synonym: "syndactyly type 2" EXACT [MONDO:0019683]
+synonym: "synpolydactyly" EXACT [Orphanet:93403]
 synonym: "synpolydactyly" EXACT [NCIT:C75003]
```

Comment:
This synonym is duplicated. The xref should be combined.

Reply:
thanks, I'll fix